<launch>

  <!-- Camera drive node -->
  <include file="$(find-pkg-share usb_cam)/launch/camera.launch.py"/>

  <!-- First image viewer node -->
  <node pkg="image_tools" exec="showimage" name="image_viwever_1">
    <remap from="/image" to="/camera1/image_raw"/>
    <param name="window_name" value="Color image window"/>
  </node>
 
  <!-- Gray scale converter node -->
  <node pkg="smarter_image_processing" exec="gray_scale_converter" name="gray_scale_converter">
    <remap from="/image" to="/camera1/image_raw"/>
  </node>
  
  <!-- Second image viewer node -->
  <node pkg="image_tools" exec="showimage" name="image_viwever_2">
    <remap from="/image" to="/camera1/gray_raw"/>
    <param name="window_name" value="Gray image window"/>
  </node>

</launch>
